### ISSUE TITLE
Random output images

### ISSUE THREAD

**Issue by miguel-fc** [thuml/predrnn-pytorch#31]
Hi, I'm using predrnn to predict future frames of experimental acquired data. To augment the data, I had cut each frame into overlapping tiles, and trained predrnn on those tiles.

Currently I'm testing how good predrnn predicts. To do so, I'm using the trained model on data, test.npz,  that had not previously seen. This data is also composed of overlapping tiles, the collection of which represents different frames.

The problem I have is this: once predictions are made,  the predicted tiles are generated in the corresponding folders, however these tiles are somewhat in a random order, or at least I'm sure they're not in the same order as in the test.npz file that was fed to the trained predrnn model. My question is, is it possible to preserve the same order? The reason for this is that I want to reconstruct frames from the predicted tiles, and if the tiles' order changes, it's very difficult to do a proper reconstruction.

Thanks
Miguel

**Comment by wuhaixu2016** [comment]
Hi, thanks for your interest.
I think you can fix the random seed or change the dataloader for data augmentation.

**Comment by miguel-fc** [comment]
Hi, Thanks. This took care of it. Appreciated it.
